src/components/views/Matches.js

```diff
@@ -121,7 +120,7 @@ const Matches = () => {
                                         key={user.otherUserId}
                                         otherUserId={user.otherUserId}
                                         status={user.status}
-                                        image={user.image} //todo
+                                        image={user.image} // was not implemented (future version?)
```

Comment:
what does it mean if it is not implemented? is the code useless? if so it should be removed entirely

Reply:
well it would be an already working implementation if we passed an image from the backend. So in a future version this would be very easy to be implemented, but ok, will remove